src/Type/Doctrine/Query/QueryResultTypeWalker.php

```diff
@@ -2028,7 +2041,7 @@ private function resolveDoctrineType(string $typeName, ?string $enumType = null,
 			$type = TypeCombinator::addNull($type);
 		}
 
-		return $type;
+			return $type;
```

Comment:
Looks like some cs check is missing :)

Reply:
this was actually *introduced* by phpcbf